docs/pages/desktop-access/getting-started.mdx

```diff
@@ -14,7 +14,7 @@ Desktop Access and log into a Windows desktop from that domain.
 This guide requires you to have:
 
 - An Active Directory domain, configured for LDAPS (Teleport requires an
-  encrypted LDAP connection)
+  encrypted LDAP connection). Typically this means installing [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
```

Comment:
I'm not sure this is a meaningful addition.

Installing AD CS is part of the setup guide, not a prerequisite.

Reply:
What's the setup guide? I only found it [in our troubleshooting documentation](https://github.com/gravitational/teleport/blob/82b001f9c33bf8d94d028d9e3e4963afc6aafb82/docs/pages/desktop-access/troubleshooting.mdx?plain=1#L184)

Reply:
🤦 you're right, ignore me. I must have been thinking about the README, not docs.

Reply:
```suggestion
  encrypted LDAP connection). Typically this means installing
  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
```